tests/e2e/specs/basic.e2e.ts

```diff
@@ -354,5 +354,5 @@ async function tryExecuteCommand(workbench: Workbench, command: string) {
 // }
 
 async function clearAllOutputs(workbench: Workbench) {
-  await tryExecuteCommand(workbench, 'notebook: clear all outputs')
+  await tryExecuteCommand(workbench, 'Notebook: Clear Outputs')
```

Comment:
```suggestion
  await tryExecuteCommand(workbench, 'Notebook: Clear Cell Outputs')
  await tryExecuteCommand(workbench, 'Notebook: Clear All Outputs')
```

I tested this locally, and noticed that (at least for me) some cell outputs lingered.

What do you think about the suggested change?